docs/tutorials/RabbitMQ_Sink.md

```diff
@@ -0,0 +1,247 @@
+:::
+
+#### Step 1.3: Verify the connector configuration
+
+Run the following command to check the configuration:
+
+::: tabs
+@tab Powershell
+```powershell:no-line-numbers
+curl -u admin:password  `
+-X GET https://your-kurrentdb-cluster-url:2113/connectors/rabbitmq-sink-quickstart/settings  
+```
+@tab Bash
+```bash:no-line-numbers
+curl -u admin:password  \ 
+-X GET https://your-kurrentdb-cluster-url:2113/connectors/rabbitmq-sink-quickstart/settings  
+```
+:::
+
+
+If successful, the response will display the current settings of your RabbitMQ Sink connector, matching your configuration file in **Step 1.1**.
+
+### Step 2: Start the RabbitMQ Sink connector
+
+Once configured, you need to start the RabbitMQ Sink connector to begin streaming events. To do this, send a `POST` request to the KurrentDB API:
+
+::: tabs
+@tab Powershell
+```powershell:no-line-numbers
+curl -i -L -u admin:password  `
+-X POST https://your-kurrentdb-cluster-url:2113/connectors/rabbitmq-sink-quickstart/start  
+```
+@tab Bash
+```bash:no-line-numbers
+curl -i -L -u admin:password  \ 
+-X POST https://your-kurrentdb-cluster-url:2113/connectors/rabbitmq-sink-quickstart/start  
+```
+:::
+
+If the RabbitMQ Sink connector starts successfully, it should return an `HTTP/1.1 200 OK` response, as shown below:
+
+```text:no-line-numbers
+HTTP/1.1 200 OK
+Content-Type: application/json; charset=utf-8
+```
+
+### Step 3: Append Events to KurrentDB
+
+To test the RabbitMQ Sink connector, first append some sample events to KurrentDB using the KurrentDB UI with the following steps:
+
+1. Log in to the KurrentDB UI.  
+2. Navigate to **Stream Browser**.  
+3. Click **Add Event**.  
+4. Enter the following details:  
+   * **Stream ID**: `LoanRequest-1`  
+   * **Event Type**: `LoanRequested`  
+   * **Event Body**:  
+      ```json
+      {
+        "Amount": 10000,
+        "loanTerm": 12
+      }
+      ```
+5. Click **Add** to append the event to the `LoanRequest-1` stream.  
+6. Repeat steps 3–5 to append more events to different streams (e.g., a stream with the **Stream ID** `LoanRequest-2`).
+
+![Append events to LoanRequest streams ](images/kafka-sink-quickstart-add-event.png =300x)
+
+
+### Step 4: Verify events in RabbitMQ
+
+Now, you can use RabbitMQ Management UI or CLI to verify that the events have been successfully streamed to RabbitMQ.
+
+#### Option 1\. Use the RabbitMQ Management UI
+
+1. Log in to the RabbitMQ Management UI (`http://your-rabbitmq-host:15672`), replacing `"your-rabbitmq-host"` with your RabbitMQ server address.  
+2. Navigate to **Exchanges**.  
+3. Find **your exchange** specified in the connector configuration.  
+4. Check the **queues bound to the exchange** and inspect messages.
+
+#### Option 2\. Use `rabbitmqctl` (Command Line)
+
+Alternatively, check the messages using the RabbitMQ CLI `rabbitmqctl`, which is automatically installed with RabbitMQ. 
+
+1. You need to start the RabbitMQ service to use `rabbitmqctl`.
```

Comment:
If the user followed the guide here so far, having the rabbitMQ service up is a guarantee, no?